### DISCUSSION TITLE
nginx

### DISCUSSION THREAD

**Discussion by sckhg1367** [open-quantum-safe/oqs-provider#418] [Q&A]

**Comment by baentsch** [comment]
Thanks for your interest using, maybe contributing to the project. We already do something very similar in the [profiling](https://github.com/open-quantum-safe/profiling) sub project, most notably [here](https://github.com/open-quantum-safe/profiling/blob/main/perf/scripts/handshakes.py). It may be easiest for you to swap out the `openssl s_server` calls there with setup calls to a PQ-enabled `nginx` (e.g., following [this code](https://github.com/open-quantum-safe/oqs-demos/tree/main/nginx)).

Also tagging @SWilson4 for further input as he volunteered to revamp the current `profiling` code.

**Comment by baentsch** [comment]
As this is no (code) problem/issue, converting to discussion.

**Reply by baentsch** [reply]
@sckhg1367 did you manage to do what you wanted? Any further input from our side or can we consider this issue closed?

**Reply by baentsch** [reply]
Thanks for closing the issue. With the deletion of the original question you are robbing future users with similar questions from the possibility to use github for searching for answers, though.
